uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-notification-badge.component.js

```diff
@@ -37,7 +37,7 @@ define([
     function fetchCount () {
       vm.loading.count = true;
 
-      return LeaveRequest.all(vm.filters)
+      return LeaveRequest.all(vm.filters, null, null, null, false)
```

Comment:
To be honest I am not sure if we will have time to implement this big of a change now, even if we agree with the proposed solution. But yeah there is no harm in starting the discussion.